optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/admin/SystemPropertiesRetriever.java

```diff
@@ -18,45 +18,57 @@
 
 import java.time.DateTimeException;
 import java.time.ZoneId;
+import java.util.Optional;
 
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
```

Comment:
This could be done with `@ConfigurationProperties` but I don't think it's important now. We can improve it later.

https://quarkus.io/guides/config-reference